### PR TITLE
Enable antialiasing prop

### PR DESCRIPTION
**Pull request by kmathew-pdftron**
antialiasing prop Monday

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -1696,6 +1696,16 @@ quads indicate the quad boundary boxes for the selection, which could have a siz
 
 ### Others
 
+#### enableAntialiasing
+bool, optional
+
+Define whether antialiasing should be applied
```
Comment (sgong-pdftron) [inline API.md:1702]: need to mention what's the default?

File: android/src/main/java/com/pdftron/reactnative/views/DocumentView.java [hunk 2]
```diff
@@ -848,6 +848,17 @@ public void setZoom(double zoom) {
         }
     }
 
+    public void setAntiAliasing(boolean enableAntialiasing) throws PDFNetException {
+        if (getPdfViewCtrl() != null) {
+            try{
```
Comment (sgong-pdftron) [inline android/src/main/java/com/pdftron/reactnative/views/DocumentView.java:853]: always format your code

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (kmathew-pdftron) [reply API.md]: yes done

File: android/src/main/java/com/pdftron/reactnative/views/DocumentView.java (hunk 2 above)
Comment (kmathew-pdftron) [reply android/src/main/java/com/pdftron/reactnative/views/DocumentView.java]: yes there is one note though I have enabled format on save so the whole document gets reformatted. Is that ok?

**Review by sgong-pdftron — APPROVED** [review]

**Review by JamieDass — APPROVED** [review]

**Pull request merged**